packages/flutter_tools/pubspec.yaml

```diff
@@ -11,8 +11,10 @@ dependencies:
   archive: 1.0.33
   args: 1.4.2
   cli_util: 0.1.2+1
+  code_builder: 3.0.3
   coverage: 0.10.0
   crypto: 2.0.2+1
+  dart_style: 1.0.10
```

Comment:
We're actually trying to reduce our dependencies currently so we probably will want to avoid adding these if we can help it.

Reply:
I can generate the code by hand but maybe it will not right formatted.

Reply:
We don't actually use exact dartfmt style either so technically it might not be right formatted either way. :-)